EMIP/1-1000/EMIP00816.xml

```diff
@@ -69,7 +261,16 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </xi:include>
       </encodingDesc>
       <profileDesc>
-         <langUsage><language ident="en">English</language></langUsage>
+        <textClass>
+            <keywords>
+                <term key="ChristianLiterature"/>
+                <term key="Homily"/>
+            </keywords>
+        </textClass>
+         <langUsage>
+           <language ident="en">English</language>
+           <language ident="gez">Gǝʿǝz</language>
+         </langUsage>
       </profileDesc>
       <revisionDesc>
          <change who="PL" when="2018-01-18">Created
```

Comment:
you should add a change statement detailing what changes you have made. You have done a lot here, so you can summarise, 'added ms items' and then give the range of numbers. 'added quire data' and give the range of information, etc. This is quite important when material is returned to after some time, so that other editors can see what you have done and why.